pkg/resources/statefulsets/statefulset.go

```diff
@@ -331,6 +343,39 @@ func NewForCluster(cluster *api.MySQLCluster, images operatoropts.Images, servic
 		})
 	}
 
+	if cluster.RequiresCustomSSLSetup() {
+		podVolumes = append(podVolumes, v1.Volume{
+			Name: mySQLSSLVolumeName,
+			VolumeSource: v1.VolumeSource{
+				Projected: &v1.ProjectedVolumeSource{
+					Sources: []v1.VolumeProjection{
+						v1.VolumeProjection{
+							Secret: &v1.SecretProjection{
+								LocalObjectReference: v1.LocalObjectReference{
+									Name: cluster.Spec.SSLSecretRef.Name,
+								},
+								Items: []v1.KeyToPath{
```

Comment:
Can we use a [TLS secret](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls)?

Reply:
If we can use it i'd like to - unfortunately we have to provide the CA cert file too for MySQL to be happy - tls secrets only have the server cert and key pair. Also it isn't tls, it's ssl still.

So we could 1) add a ca.crt field to the tls secret. 2) have the ca.crt in a different secret or 3) leave it how it is.

Thoughts?

Reply:
Ah, shame. I’d follow the naming scheme from the TLS secret and add the `ca.crt` as you suggest.

Reply:
done